ACT-reqs.md

```diff
@@ -4,64 +4,68 @@ ACT Framework Requirements
 Abstract
 --------
 
-The Accessibility Conformance Testing Framework will be a W3C Recommendation, developed by the ACT Taskforce, lead by the WCAG Working Group. This document outlines the requirements for the ACT Framework. The ACT Framework is one part of the deliverables of the ACT Taskforce, for more details on this read [[ACT Deliverables]].
+The Accessibility Conformance Testing (ACT) Framework is intended to become a W3C Recommendation. It is developed by the [WCAG Working Group](https://www.w3.org/WAI/GL/) through its [ACT Task Force](https://www.w3.org/WAI/GL/task-forces/conformance-testing/). This document outlines the requirements for the ACT Framework. An overview of ACT efforts including complimentary deliverables is described in [What is ACT](https://www.w3.org/WAI/GL/task-forces/conformance-testing/wiki/ACT_Overview_-_What_is_ACT).
 
 Status of This Document
 -----------------------
 
-This is the first draft of this document. It has not yet been submitted for formal review to the WCAG Working Group.
+This is the first draft of the ACT Framework Requirements document. It has not yet been reviewed or approved by the WCAG Working Group. At this stage, input on the following aspects is requested:
+
+- Completeness: Are there missing requirements?
+- Clarity: Are the described requirements clear?
+- Cohesion: Are there requirement contradictions?
```

Comment:
Not hung on these particular words, but always good to give the reader specific questions to think about during their review.